.changeset/red-clouds-sit.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": minor
+---
+
+Added: `ignoreAtRules:[]` regex support in `at-rule-empty-line-before`
```

Comment:
We have 2 precedents, either

```suggestion
Added: regex support to `ignoreAtRules` option of `at-rule-empty-line-before`
```
or
```suggestion
Added: regex support for `ignoreAtRules` option of `at-rule-empty-line-before`
```

Whichever you choose remember to fix the title too.